main.go

```diff
@@ -44,20 +43,27 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+const (
+	default_namespace = "default"
+)
+
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(nodemaintenancev1beta1.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
 }
 
+var leaderElectionNamespace string
+
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+	flag.StringVar(&leaderElectionNamespace, "leader-elect-namespace", default_namespace, "The leader election namespace")
```

Comment:
Since default_namespace var is used only in one place I don't think it's necessary to define a const for it - same way that you don't define a const for each string those flags consume.